packages/dialtone-tokens/tokens/$metadata.json

```diff
@@ -16,9 +16,7 @@
     "components/badge/dark",
     "components/button/default",
     "components/checkbox/default",
-    "components/checkbox/dark",
```

Comment:
This and others like it below is intentional. With the border color update made to default, its same values apply to dark too.